introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone_dist_script_mode.ipynb

```diff
@@ -4,18 +4,24 @@
    "cell_type": "markdown",
```

Comment:
<p>Numbering in Contents is off (two "1." in a sublist)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Looks normal (A, B subheadings) in the GitHub preview: <a href="https://github.com/aws/amazon-sagemaker-examples/blob/702a3d0420b57a8a46b72605fa62005da2aace95/introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone_dist_script_mode.ipynb" target="_blank">https://github.com/aws/amazon-sagemaker-examples/blob/702a3d0420b57a8a46b72605fa62005da2aace95/introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone_dist_script_mode.ipynb</a></p>